aio/content/navigation.json

```
@@ -658,6 +658,11 @@
               "url": "guide/ivy-compatibility",
               "title": "Ivy Compatibility Guide",
               "tooltip": "Details to help you make sure your application is compatible with Ivy."
+            },
+            {
+              "url": "guide/migration-legacy-message-id",
+              "title": "Migration: Legacy Localization IDs",
+              "tooltip": "Learn how to migrate old IDs for localization to new, more stable ones."
```

Let's keep drumming this terminology home.

```suggestion
              "tooltip": "Learn how to migrate old legacy IDs for localization to new, more stable ones."
```